java.lang.ExceptionInInitializerError Caused by: java.lang.NullPointerException: in == null   at java.util.Properties.load(Properties.java:246)

I tried adding leak canary to our app and we get this crash:

    java.lang.ExceptionInInitializerError
      at org.eclipse.mat.hprof.HprofIndexBuilder.fill(HprofIndexBuilder.java:54)
      at org.eclipse.mat.parser.internal.SnapshotFactory.parse(SnapshotFactory.java:193)
      at org.eclipse.mat.parser.internal.SnapshotFactory.openSnapshot(SnapshotFactory.java:106)
      at com.squareup.leakcanary.HeapAnalyzer.openSnapshot(HeapAnalyzer.java:134)
      at com.squareup.leakcanary.HeapAnalyzer.checkForLeak(HeapAnalyzer.java:87)
      at com.squareup.leakcanary.internal.HeapAnalyzerService.onHandleIntent(HeapAnalyzerService.java:56)
      at android.app.IntentService$ServiceHandler.handleMessage(IntentService.java:65)
      at android.os.Handler.dispatchMessage(Handler.java:102)
      at android.os.Looper.loop(Looper.java:145)
      at android.os.HandlerThread.run(HandlerThread.java:61)
    Caused by: java.lang.NullPointerException: in == null
      at java.util.Properties.load(Properties.java:246)
      at org.eclipse.mat.util.MessageUtil.(MessageUtil.java:28)
      at org.eclipse.mat.util.MessageUtil.(MessageUtil.java:13)
      ... 10 more

--- a/library/leakcanary-analyzer/build.gradle
+++ b/library/leakcanary-analyzer/build.gradle
@@ -14,7 +14,7 @@ repositories {
 }
 
 dependencies {
-  compile 'com.squareup.haha:haha:1.1'
+  compile 'com.squareup.haha:haha:1.2'
   compile project(':leakcanary-watcher')
   testCompile 'junit:junit:4.12'
 }

